PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.java

```diff
@@ -108,12 +109,11 @@ String createRequestBody(
 
         parameters.putOpt(PAYER_EMAIL_KEY, userAuthenticationEmail);
 
-        if (enablePayPalAppSwitch) {
+        if (enablePayPalAppSwitch && appLink != null && !appLink.isEmpty()) {
             parameters.put(ENABLE_APP_SWITCH_KEY, enablePayPalAppSwitch);
             parameters.put(OS_VERSION_KEY, Build.VERSION.SDK_INT);
```

Comment:
General question, is this our SDK version or the merchants OS version of their device?

Reply:
According to the docs: _The SDK version of the software currently running on this hardware device._